packs/spells/spellmasters-ward.json

```diff
@@ -11,7 +11,7 @@
         "damage": {},
         "defense": null,
         "description": {
-            "value": "<p><strong>Duration</strong> sustained up to 1 minute</p>\n<hr />\n<p>You create one or more glowing protective runes that revolve around you. For each additional action you use when Casting the Spell, increase the number of runes you create by one, to a maximum of three runes for 3 actions.</p>\n<p>When you next take damage from a spell, a rune is expended to prevent @Damage[2d6[damage]] damage. If damage from the triggering spell remains, additional runes are expended until the damage is completely negated, or until all runes have been spent, whichever comes first.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> Each rune prevents an additional @Damage[1[damage]] damage.</p>"
+            "value": "<p><strong>Duration</strong> sustained up to 1 minute</p>\n<hr />\n<p>You create one or more glowing protective runes that revolve around you. For each additional action you use when Casting the Spell, increase the number of runes you create by one, to a maximum of three runes for 3 actions.</p>\n<p>When you next take damage from a spell, a rune is expended to prevent [[/r 2d6+max(0,@item.level -5)]] of that spell's damage. If damage from the triggering spell remains, additional runes are expended until the damage is completely negated, or until all runes have been spent, whichever comes first.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> Each rune prevents an additional 1 damage.</p>"
```

Comment:
```suggestion
            "value": "<p><strong>Duration</strong> sustained up to 1 minute</p>\n<hr />\n<p>You create one or more glowing protective runes that revolve around you. For each additional action you use when Casting the Spell, increase the number of runes you create by one, to a maximum of three runes for 3 actions.</p>\n<p>When you next take damage from a spell, a rune is expended to prevent [[/r 2d6 + @item.rank - 5)]] of that spell's damage. If damage from the triggering spell remains, additional runes are expended until the damage is completely negated, or until all runes have been spent, whichever comes first.</p>\n<hr />\n<p><strong>Heightened (+1)</strong> Each rune prevents an additional 1 damage.</p>"
```